ecommerce/certprep/static/templates/_offer_error.html

```diff
@@ -0,0 +1,4 @@
+<div class="depth depth-2 message-error-content offer-error">
```

Comment:
We cannot verify the changes you have made to this file, please separate the original template from your changes. @sergivalero20

Reply:
I removed this file because I determined that it wasn't necessary.